pkg/eval/external_cmd.go

```diff
@@ -28,6 +28,10 @@ type ExternalCmd struct {
 	Name string
 }
 
+// TODO: Decide what `kind-of` should report for an external command object
+// and document the rationale for the choice. It's not obvious why this
+// returns "fn" rather than "external". If this is changed
+// TestBuiltinFnExternal will need to be updated.
```

Comment:
The entire "kind" mechanism is kind of half-baked, so calling it out here is a bit weird. Better remove this and document the situation in vals.Kind instead.

Reply:
Done. Comment moved (albeit rewritten) to the doc string for `func Kind`.